gitlint-core/gitlint/contrib/rules/block-fixup-squash.py

```diff
@@ -0,0 +1,14 @@
+from gitlint.rules import CommitRule, RuleViolation
+
+class BlockFixupSquash(CommitRule):
+    name = "block-fixup-squash"
+    id = "BF1"
```

Comment:
Following existing naming conventions:
```suggestion
class DisallowFixupSquashCommit(CommitRule):
    name = "contrib-disallow-fixup-squash"
    id = "CC2"
```

Please also update filename to match new classname.